rpc/cc/arduino/cli/commands/v1/common.proto

```diff
@@ -126,3 +129,9 @@ message Profile {
   // FQBN specified in the profile.
   string fqbn = 2;
 }
+
+message Online {
+  // A URL provided by the author of the platform's package, intended to point
+  // to their online help service.
+  string online = 1;
+}
```

Comment:
It sounds a bit weird to call this message `Online`, what about a more self-explanatory name like `Help` or better `HelpResources`?

Reply:
I'm suggesting to rename the structure but not the field, for example:

```
message HelpResources {
  // A URL provided by the author of the platform's package, intended to point
  // to their online help service.
  string online = 1;
}
```

This "message" has the same `Online` field that you can access via `obj.Help.Online`.

Reply:
Oh, I misunderstood what you meant. I'll modify it as suggested.